packages/database/scripts/create-database.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script creates a new database in the mysql container.
+# This is only needed for local development.
+
+read -p "What is the MySQL Docker container name? [mysql]: " container_name
+container_name=${container_name:-mysql}
+
+read -p "What is the new database name? [datahub_development]: " database_name
+database_name=${database_name:-datahub_development}
+
+read -p "What is the mysql username? [root]: " user_name
+user_name=${user_name:-root}
+
+read -p "What is the mysql password?: " password
+
+docker exec ${container_name} mysql --user=${user_name} --password=${password} -e "CREATE DATABASE ${database_name};" && 
```

Comment:
Should we quote the variables (in case the vars hold sensitive characters, e.g. quotes), e.g.:

`--user="${user_name}"`?